### ISSUE TITLE
The test with the landmark.py you applied is not accurate.

### ISSUE THREAD

**Issue by XiaXuehai** [luoyetx/deep-landmark#16]
I use the landmark and the caffemodel you applied to test the image,and the result wasn't  accurate.
and I see the `classification.cpp` caffe applied.The input testing image should minus mean_file.Do you think it's a factor of inaccuracy.

**Comment by luoyetx** [comment]
The code [here](https://github.com/luoyetx/deep-landmark/blob/master/webapp/landmark.py#L171) process the image. The model here is still not comparable to the original paper.

**Comment by XiaXuehai** [comment]
THX. But the mean_file in caffe's example is the mean of the training image.And it's not  the same as  the following:

```
def _processImage(self, imgs):
        imgs = imgs.astype(np.float32)
        for i, img in enumerate(imgs):
            m = img.mean()
            s = img.std()
            imgs[i] = (img - m) / s
        return imgs
```

Does it affect the result?

**Comment by luoyetx** [comment]
@XiaXuehai Yes. Caffe support an external mean_file for data layer. But I didn't use it. Actually, I did the image processing in my own code and give a raw data input to the network.

**Comment by hugowangyj** [comment]
Hello, luoyetx,
   what's your lowest loss on the test dataset? I find mine is a large gap with the paper.

**Comment by luoyetx** [comment]
@hugowangyj You can find some data in this [pdf](https://github.com/luoyetx/face-alignment-presentation/blob/master/Face-Alignment.pdf).

**Comment by hugowangyj** [comment]
Yes, but it is still a large lag behind the original paper.
Have you the latest results?
